go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/sublime-security/go-import-name-validator
 
-go 1.18
+go 1.19
```

Comment:
Since this package isn't opting in to any 1.19 features, I think it makes more sense to leave this as 1.18 so it's compatible with both!

Reply:
Since there isn't any CI that is validating formatting, that makes sense. I'll just go ahead and close this PR then!